scripts/cli.sh

```diff
@@ -1,21 +1,28 @@
 set -e
 
+CLI_PATH=$ACTION_WORKSPACE/keg-cli
+
 clone_cli () {
   echo "==== Cloning Keg-CLI branch $INPUT_CLI_GIT_BRANCH"
   git -C $ACTION_WORKSPACE clone --single-branch --branch $INPUT_CLI_GIT_BRANCH https://github.com/simpleviewinc/keg-cli.git
 }
 
 install_cli () {
   echo "==== Installing Keg-CLI dependencies..." 
-  cd $ACTION_WORKSPACE/keg-cli
+  cd $CLI_PATH
```

Comment:
This comment is for line 13 =>. `yarn install`
* The install command gets called in the `scripts/ci/setupKegEnvironment.sh` of the `keg-cli`
* If you look at the workflow logs, you'll see that yarn install is called twice
* We should remove this call and let the `setupKegEnvironment.sh` script handle it